staff/RominaFior/js/challenges/count-words/scripts/main.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var count= countWords('Hello world')
+
+console.log('countWords ("hello world" should return 2) ', count === 2, count);
+
+
+try {
+    convert = toCamelCase(true)
```

Comment:
Shouldn't 'convert' be declared like this `var convert` before the first try/catch? 🤔

Reply:
indeed! thank you Alba, now it shoud work!